app/jobs/finance/statements/mark_as_payable.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Finance
+  module Statements
+    class MarkAsPayable < ApplicationJob
+      def perform
+        Finance::Statement.where(deadline_date: 1.day.ago.to_date).find_each do |statement|
+          statement
+            .participant_declarations
+            .where
+            .not(state: ParticipantDeclaration.states.values_at(:voided, :ineligible))
```

Comment:
can we whitelist instead of blacklist here? as in the future there will be new states and it will cause a bug here